.github/workflows/rust.yml

```diff
@@ -65,7 +64,9 @@ jobs:
       run: rustup target add riscv32imc-unknown-none-elf --toolchain nightly-2023-01-03-x86_64-unknown-linux-gnu
     - name: Install stdlib
       run: rustup component add rust-src --toolchain nightly-2023-01-03-x86_64-unknown-linux-gnu
+    - name: Install pilcom
+      run: git clone https://github.com/0xPolygonHermez/pilcom.git  && cd pilcom && npm install
     - name: Build
       run: cargo build --release
     - name: Run tests
-      run: cargo test -r --verbose -- --ignored --nocapture
+      run: PILCOM=$(pwd)/pilcom/ cargo test -r --verbose -- --ignored --nocapture --test-threads=1
```

Comment:
Why can't we use multithreading here? Because of memory constraints? Can you add a comment if it is that?

Reply:
Yeah, it is that. Looking through GitHub action docs, it seems we have 7 GB to run it, so it only fits one thread running PILCOM. Trying with the default number of threads (that I assume is 2, per the docs) get the processes terminated (by the OOM killer, I assume).